app/renderer/components/Inspector/HighlighterCentroid.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react';
+import InspectorCSS from './Inspector.css';
+
+/**
+ * Shows all element centroids
+ */
+export default class HighlighterCentroid extends Component {
+
+  render () {
+    const {selectedElement = {}, selectHoveredElement, unselectHoveredElement, hoveredElement = {}, selectElement, unselectElement, element, elementProperties,
+           currentRender, hoveredCentroid, selectCentroid, unselectCentroid, selectedCentroid, unselectHoveredCentroid, selectHoveredCentroid,
+           RENDER } = this.props;
+    const {path: hoveredPath} = hoveredElement;
+    const {path: selectedPath} = selectedElement;
+    const { centerX, centerY, keyCode, path, container } = elementProperties;
+    const key = `C:${path}`;
+
+    const centroidClasses = [InspectorCSS['centroid-box']];
+    centroidClasses.push(InspectorCSS[currentRender]);
+
+    // Highlght specific centroids based off of what element user clicks
+    if (currentRender !== RENDER.EXPAND) {
+      if (hoveredPath === path) {
+        centroidClasses.push(InspectorCSS['hovered-element-box']);
+      }
+      if (selectedPath === path) {
+        centroidClasses.push(InspectorCSS['inspected-element-box']);
+      }
+    }
+
+    if (currentRender !== RENDER.CENTROID) {
+      if (hoveredCentroid === keyCode) {
+        centroidClasses.push(InspectorCSS['hovered-element-box']);
+      }
+      if (selectedCentroid === keyCode && !element) {
+        centroidClasses.push(InspectorCSS['inspected-element-box']);
+      }
+    }
+
+    const divStyle = {
```

Comment:
I removed the base options and added it into the css itself. However, I can't add this object to the top of the file just because I have to access certain properties like `keyCode`, `selectedCentroid`, `element`, `centerX`, `centerY`, and etc.